tests/decap/test_decap.py

```diff
@@ -203,6 +205,8 @@ def test_decap(tbinfo, duthosts, ptfhost, setup_teardown, mux_server_url,
             wait(30, 'Wait some time for mux active/standby state to be stable after toggled mux state')
 
         log_file = "/tmp/decap.{}.log".format(datetime.now().strftime('%Y-%m-%d-%H:%M:%S'))
+        if skip_traffic_test:
```

Comment:
Can this condition be moved up before the condition that waits for 30 seconds for dualtor ? Does the mux active/standby state not being stable applicable to `vs` ?

Reply:
Good point, currently we are only adding it to t0 test, so it's no effect, but skip early would be better if we add it to KVM dualtor test in the future